card-style.cls

```diff
@@ -36,6 +37,27 @@
 
 \newcounter{cards}
 
+\newcommand{\CardType}[1]{%
+  \def\cardcolor{}
+  \ifnum\pdfstrcmp{#1}{Item}=0      \def\cardcolor{blue} \fi
+  \ifnum\pdfstrcmp{#1}{Action}=0    \def\cardcolor{yellow} \fi
+  \ifnum\pdfstrcmp{#1}{Spell}=0     \def\cardcolor{green} \fi
+  \ifnum\pdfstrcmp{#1}{Cantrip}=0   \def\cardcolor{green} \fi
+  \ifnum\pdfstrcmp{#1}{Focus}=0     \def\cardcolor{green} \fi
+  
+  \ifnum\pdfstrcmp{\cardcolor}{}=0 \else
+    \tikz[remember picture,overlay] {%
+      \draw [color=\cardcolor,line width=2mm]
+        (current page.south west)
+        rectangle
+        (current page.north east);
+    }
+  \fi
```

Comment:
round corners?

Reply:
Yes, I was still in the phase of getting something working at all 😃